Assets/Editor/ActionMasterTest.cs

```diff
@@ -1,295 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using NUnit.Framework;
-using UnityEngine;
-
-[TestFixture]
-public class ActionMasterTest
-{
-
-	private ActionMaster.Action endTurn = ActionMaster.Action.EndTurn;
-	private ActionMaster.Action midFrameReset = ActionMaster.Action.MidFrameReset;
-	private ActionMaster.Action frameReset = ActionMaster.Action.FrameReset;
-	private ActionMaster.Action endGame = ActionMaster.Action.EndGame;
-	private ActionMaster actionMaster;
-	private Frame currentFrame;
-	
-	[SetUp]
-	public void Setup()
-	{
-		actionMaster = new ActionMaster();
-	}
-
-	[Test]
-	public void PassingTest () {
-		Assert.AreEqual (1, 1);
-	}
-
-	[Test]
-	public void T01OneStrikeReturnsEndTurn()
-	{
-		Frame.ResetFrames();
-		int[] rolls = {10};
-		for (int i= 0; i < rolls.Length; i++)
-		{
-			if (Frame.FrameList.Count < 1 || Frame.FrameList.Last().IsFrameComplete())
-			{
-				currentFrame = new Frame(rolls[i]);
-			}
-			else
-			{
-				currentFrame.AddRoll(rolls[i]);
-			}	
-		}	
-		Assert.AreEqual(endTurn, actionMaster.NextAction(currentFrame));
-	}
-
-	[Test]
-	public void T02Bowl8ReturnsTidy()
-	{
-		Frame.ResetFrames();
-		int[] rolls = {8};
-		for (int i= 0; i < rolls.Length; i++)
-		{
-			if (Frame.FrameList.Count < 1 || Frame.FrameList.Last().IsFrameComplete())
-			{
-				currentFrame = new Frame(rolls[i]);
-			}
-			else
-			{
-				currentFrame.AddRoll(rolls[i]);
-			}	
-		}	
```

Comment:
As this for loop appears verbatim in every single test, it's best practice to "refactor" it so that every test uses the same code. That way:
* There are less chances that errors can be introduced in the tests by human error
* It's clear to the reader that every single version is the same, and avoids having to read through each test thoroughly looking for differences
* Future tests are easier and faster to write